packages/core/src/Stamp/Stamp.stories.args.ts

```diff
@@ -10,4 +10,9 @@ export const argTypes = {
       type: 'select',
     },
   },
+  variation: {
+    type: { name: 'string' },
+    options: ['fill', 'outline', 'solid'],
```

Comment:
Good thing I noticed this in the migration guide re `solid`! 
<img width="1056" alt="CleanShot 2022-12-14 at 13 41 29@2x" src="https://user-images.githubusercontent.com/3260642/207684101-0693f33a-315e-425b-9563-920afb7a8d59.png">